install.sh

```diff
@@ -16,21 +16,21 @@ amd_lshw() {
   lshw -c display -numeric -disable network | grep -q 'vendor: .* \[1002\]'
 }
 
-gpu_check() {
-  if available lspci && lspci -d '10de:' | grep -q 'NVIDIA'; then
-    nvidia_available="true"
-  elif available lshw && nvidia_lshw; then
-    nvidia_available="true"
-  elif available nvidia-smi; then
-    nvidia_available="true"
-  fi
+# gpu_check() {
+  # if available lspci && lspci -d '10de:' | grep -q 'NVIDIA'; then
+  #  nvidia_available="true"
+  # elif available lshw && nvidia_lshw; then
```

Comment:
This line looks wrong.

Shouldn't this be 
if available nvidia_lshw && nvidia_lshw; then?

Reply:
nvidia_lshw is a bash function a couple of lines up

lshw is a binary required to query for GPU type, not always installed.

Reply:
Thanks.